Makefile

```diff
@@ -62,6 +62,7 @@ build/llvm.BUILT:
 		-DLLVM_STATIC_LINK_CXX_STDLIB=ON \
 		-DLLVM_HAVE_LIBXAR=OFF \
 		-DCMAKE_OSX_ARCHITECTURES="arm64;x86_64" \
+		$(LLVM_CMAKE_FLAGS) \
```

Comment:
Maybe put this add the of the command line (line 77)?

Reply:
Do you mean the bottom of the command line? Sure, will do after the current pipeline finishes.